README.md

```diff
@@ -278,7 +272,12 @@ through the gate, either because:
 * they're on the list
 * they're part of the first _sample_ users
 
-Sampling is done by checking the character with which _user_ begins. For example,
+Sampling is done by hashing the _user_ string and projecting it onto a sample
+space, by converting it to an unsigned 32 bit integer, and then normalizing by
+the size of the max UInt32 (essentially [this solution](https://stats.stackexchange.com/a/70884), but with only two
+buckets ("matches", "doesn't match").
```

Comment:
I think this is a bit more technical than the README requires. I don't even think an explanation is even strictly required now that it no longer will accompany `sampleCharacterSet` (that bit below should be removed right?) The important things to stress are that sampling is both deterministic and uniformly random.

Reply:
https://github.com/mixmaxhq/user-gate/pull/5/commits/d885aa229936ac4ca262078cd0190e6bb1ae24a0